### ISSUE TITLE
Bug: not populating maxitems in ah (despite increasing it in sql file or via gm command)

### ISSUE THREAD

**Issue by Nottify15** [azerothcore/mod-ah-bot#106]
### Current Behaviour

Hi, I've recently installed ahbot module.
I don't know if it's a bug or an error I made (i think the second one) but I don't know how to resolve it.
I changed the number of maxitems in sql file to 25000 but I see only 249 items in game... I certainly did something wrong but I can't figure it out... 
Thanks for your help
![ahbot numberitems](https://github.com/user-attachments/assets/d17bd042-1093-4516-92c5-f1a31cee7185)
![sql modahbot](https://github.com/user-attachments/assets/28416d10-74de-4f6a-93a3-4927f7723393)

### Expected Behaviour

Should have the proper number of autions regarding sql/maxitems config

### Steps to reproduce the problem

.

### AC rev. hash/commit

.

### Operating system

w10x64

### Custom changes or Modules

individual progression

**Comment by kewinrausch** [comment]
Greetings,

For performance reason the module does not introduce all the items at once, but operate by adding up to `ItemsPerCycle` each time the auction house has to update (actually it perform `ItemsPerCycle` attempts to add items). This means every minute less or more you shall see something entering the market, depending on your configuration (if you have whitelist enabled, or if it's configured to not add more than X stacks of the same object with `DuplicatesCount`).

In addition the configuration is read at startup, so you will have to reset the server or reload the configuration to allow the module to read again the sql tables and load the data from the database into memory.

To sum up:

- Have you set up the max items configuration after the server start? If yes, restart the server or reload the configuration with `.reload config`
- Did the problem persist also after waiting some time (like an hour or so)?
- Can you please share your configuration file and situation of `mod_auctionhousebot`?

**Comment by Nottify15** [comment]
Hi kewinrausch, 

Thanks for your detailed answer :)

I played a bit but seen no change in ah auctions (always stuck at 249 items). I tried to reload config in worldserver but nothing changes.
I noticed that my edit in realmlist (timezone, pvp, ... in heidisql) are not saved either.
[configs.zip](https://github.com/user-attachments/files/17006863/configs.zip)

I'm pretty new to this. I try to do my best but i'm kind of lost here...

**Comment by AldebaraanMKII** [comment]
I`m encountering a similar issue.
I created a account and character to be used by the module. I set the max auctions to be 8000/8000/12000 for alliance/horde/neutral respectively, but after over half a hour no auctions are found in the AH.

**Comment by kewinrausch** [comment]
Ok, everything seems fine, and from what I get now it's working.

Maybe there was something still pending on the database side and the game was working with old settings; I don't know heidisql, I have a different setup with my server. To increase the number of items what you have to do is play with `maxitems` and the `ItemsPerCycle` configurations.

If you are good with it now you can close this issue.

**Comment by Trus3683** [comment]
I am having the same issue. I raised min items / 49000 and max items / 50000 in the sql database a few days ago - after several restarts and many hours of play, the amount of items seems stagnant at around 24900.

**Comment by Routhinator** [comment]
Make sure you check the number of items being discovered, and then multiply that by the duplicate stacks limit if enabled. If you set the duplicate stacks to 5, for example - and you're at the default number of items of ~12800 - then the maximum possible number of auctions is 61900 - so you'd never get more than that no matter what you set minimum to.
